### ISSUE TITLE
Why "use" and "replace" suggestions?

### ISSUE THREAD

**Issue by ensignavenger** [dosisod/refurb#17]
Neat package!  Thank you for making it and proving it under an open source license!

In the example, most of the suggestions were in the style and pattern of "use x instead of y", but one of them was "replace x with y".   

I am wondering if there is a reason why these messages are different and if so, what.  I think having a consistant pattern for how the information is presented would be wise, but perhaps there is a reason for the difference I am not seeing?

**Comment by dosisod** [comment]
Thank you! I love open source, and it would be a shame to not share this project with the world!

I agree that consistent wording in the error messages is a must. I did this mostly subconsciously, but if I had to guess, the "Use y instead of x" has more words, and mentions the replacement first instead of the current line, whereas "Replace x with y" is less words, and tells you what needs to change first, and what to replace it with after.

In short, there is no explicit reason why they are different, I just decided to word it like that because it felt better in that scenario, and probably got too lazy to change the rest of them. I prefer the "Replace x with y" form better, so I will update the messages to reflect that.

Thank you for reporting this!
